src/common/locations.ts

```diff
@@ -1,11 +1,20 @@
 /** Helpers for dealing with the State / Counties dataset. */
 import US_STATE_DATASET from 'components/MapSelectors/datasets/us_states_dataset_01_02_2020.json';
-import { each, sortBy, takeRight, has } from 'lodash';
 import urlJoin from 'url-join';
+import {
+  each,
+  sortBy,
+  takeRight,
+  has,
+  partition,
+  isUndefined,
+  toLower,
+} from 'lodash';
 import { assert } from './utils';
 import countyAdjacencyMsa from './data/county_adjacency_msa.json';
 import collegesByFips from './data/colleges_by_fips.json';
 import { REVERSED_STATES } from 'common';
+import { isCounty } from '../components/AutocompleteLocations';
```

Comment:
FWIW, I'd be a fan of moving `isCounty` (and probably `isState` and `belongsToState`) into this file.

Reply:
moved!